{"uid":"99967008f265d2af","name":"Verificar apresentação do histórico dos dados de diretrizes diárias","fullName":"Tela Inicial: Verificar apresentação do histórico dos dados de diretrizes diárias","historyId":"435ff56c9e963fcddf22359e7b27324d","time":{"start":1747764746841,"stop":1747764748287,"duration":1446},"status":"failed","statusMessage":"AssertionError: Gráfico 'Histórico Diretrizes' não encontrado.\n","statusTrace":"  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 121, in step_validar_graficos_historico_diretriz\n    page.validar_graficos_historico_diretriz()\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 198, in validar_graficos_historico_diretriz\n    assert historico, \"Gráfico 'Histórico Diretrizes' não encontrado.\"\n           ^^^^^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Gráfico 'Histórico Diretrizes' não encontrado.\n","statusTrace":"  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 121, in step_validar_graficos_historico_diretriz\n    page.validar_graficos_historico_diretriz()\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 198, in validar_graficos_historico_diretriz\n    assert historico, \"Gráfico 'Histórico Diretrizes' não encontrado.\"\n           ^^^^^^^^^\n","steps":[{"name":"Given que eu estou na Página Inicial","time":{"start":1747764746841,"stop":1747764746993,"duration":152},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When eu clico no botão \"Ver Histórico\" para diretrizes diárias","time":{"start":1747764746994,"stop":1747764747322,"duration":328},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Then eu sou direcionado para a tela de histórico da diretriz diária","time":{"start":1747764747322,"stop":1747764748072,"duration":750},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When eu devo poder visualizar dados históricos de maneira gráfica","time":{"start":1747764748072,"stop":1747764748286,"duration":214},"status":"failed","statusMessage":"AssertionError: Gráfico 'Histórico Diretrizes' não encontrado.\n","statusTrace":"  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 121, in step_validar_graficos_historico_diretriz\n    page.validar_graficos_historico_diretriz()\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 198, in validar_graficos_historico_diretriz\n    assert historico, \"Gráfico 'Histórico Diretrizes' não encontrado.\"\n           ^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},{"name":"When usuário retorna a tela inicial","time":{"start":1747764748287,"stop":1747764748287,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":5,"attachmentStep":false,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Tela Inicial"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"99967008f265d2af.json","parameterValues":[]}